CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## Unreleased
+
+* Add `--tag` global flag, which overrides image tag part temporarily.
+  A typical use of `--tag` flag is to synchronize image tags with the tag of
+  VCSs. In order to be accessible from build hooks, Crane sets a value of
+  `--tag` to the `CRANE_TAG` environment variable. You can also set a default
+  value of `--tag` by the `CRANE_TAG` environment variable.
+
+  Example:
+  If you specify a `--tag rc-2`, you will get these results:
+
+  |original image name|overridden image name|
+  |-------------------|---------------------|
+  |nginx              |nginx:rc-2           |
+  |nginx:1.9          |nginx:rc-2           |
+  |repo/nginx         |repo/nginx:rc-2      |
+  |host:5000/nginx    |host:5000/nginx:rc-2 |
```

Comment:
Do you want to add your GitHub handle here like I did for the other releases?

Also, I added the unreleased header to master so that all PR's can use that. Mind rebasing?